tools/scripts/docs-website/create-docs-pr.js

```diff
@@ -141,36 +148,54 @@ function validateTag (version, force) {
 async function getReleaseNotes (version, releaseNotesFile) {
   console.log('Retrieving release notes from file: ', releaseNotesFile)
 
-  const data = await readReleaseNoteFile(process.cwd() + '/' + releaseNotesFile)
+  const data = await fs.readFile(process.cwd() + '/' + releaseNotesFile, 'utf8')
```

Comment:
Just want to confirm, does `fs-extra`'s `readFile` return a Promise?  Regular `fs` does not.

Reply:
As an aside, I'm not a huge fan of calling fs-extra "fs" because if i can't immediately tell that `fs-extra` is being used, it can cause some confusion when reviewing raw code.

Reply:
Yes, with fs-extra, "all fs methods return promises if the callback isn't passed."